apps/meteor/server/methods/OEmbedCacheCleanup.ts

```diff
@@ -21,12 +28,10 @@ Meteor.methods<ServerMethods>({
 			});
 		}
 
-		const date = new Date();
-		const expirationDays = settings.get<number>('API_EmbedCacheExpirationDays');
-		date.setDate(date.getDate() - expirationDays);
-		await OEmbedCache.removeAfterDate(date);
+		await executeClearOEmbedCache();
 		return {
 			message: 'cache_cleared',
+			params: [],
```

Comment:
If this `params` is needed, pls add it to the return type of method

Reply:
Good catch, done!
Thanks